src/docs/stan-reference/distributions.tex

```diff
@@ -1165,8 +1191,8 @@ \subsubsection{Stan Functions}
 \end{description}
 %
 \begin{description}
-\fitem{real}{exp\_mod\_normal\_rng}{real \farg{mu}, real \farg{sigma},
-real \farg{lambda}}{Generate a exponentially modified normal variate
+\fitem{real}{exp\_mod\_normal\_rng}{reals \farg{mu}, reals \farg{sigma},
```

Comment:
This is the thing where the return type needs to be something more generic.  It's not going to be `real` unless both arguments are `real`.  So I was suggesting something like a generic `R` and a link to the section describing RNG vectorization in the doc.